debug.log

```diff
@@ -0,0 +1,2 @@
+[1211/221328.466:ERROR:directory_reader_win.cc(43)] FindFirstFile: ������� �� ������� ����� ��������� ����. (0x3)
```

Comment:
do we need debug.log file?

Reply:
No, it came by itself. I deleted this file.